lib/db/cursor.php

```diff
@@ -146,7 +146,7 @@ function findone($filter=NULL,array $options=NULL,$ttl=0) {
 	function paginate(
 		$pos=0,$size=10,$filter=NULL,array $options=NULL,$ttl=0) {
 		$total=$this->count($filter,$ttl);
-		$count=ceil($total/$size);
+		$count=(($size!=0)?ceil($total/$size):0);
```

Comment:
Shorter alternatives because of reasons:
* `$count=$size==0?:ceil($total/$size);`
* `$count=$size?ceil($total/$size):0;`